docs/application/dotnet/guides/sensors/device-sensors.md

```diff
@@ -1,31 +1,36 @@
+---
+keyword: sensor, accel, magnet, gyro, gravity, heart rate, humidity, light, orientation, pedometer, pressure, proximity, rotation vector, motion, sleep, temperature, ultraviolet
+---
+
 # Device Sensors
 
 You can manage the sensors in the device and receive data from them.
 
-Using the Tizen.Sensor namespace, you can [create sensor instances](#sensor), which correspond to physical or software-defined sensor devices. The sensor instance is used to check the availability of and control the corresponding sensor.  
+Using the [Tizen.Sensor](/application/dotnet/api/TizenFX/latest/api/Tizen.Sensor.html) namespace, you can [create sensor instances](#sensor), which correspond to physical or software-defined sensor devices. The sensor instance is used to check the availability of and control the corresponding sensor.  
 
 After you have created a sensor instance for a specific sensor and [subscribed to sensor events](#subscribe), you can monitor the device's internal sensors for sensor value changes. The application can receive the sensor data only when the data is modified.
 
-When running an application on the emulator, you can use the Emulator Control Panel to simulate sensor data for the application.
+When running an application on the emulator, you can use Emulator Control Panel to simulate sensor data for the application.
 
 A device can have various physical and virtual sensors. The following table lists the sensors supported by Tizen.
 
 
-> **Note**   
+> [!NOTE]
 > Not all devices support all sensors, so each sensor is not necessarily available on all devices. You can [check whether a sensor is supported](#sensor).
 
 **Table: Supported sensor types**
 
 | Supported sensor types                   |                                          |                                       |
 |----------------------------------------|----------------------------------------|-------------------------------------|
-| [Accelerometer](#accelerometer)  | [Heart Rate Monitor LED Green Batch Sensor](#hrm_green_batch) | [Proximity Sensor](#proximity)   |
-| [AutoRotation Sensor](#auto_rotation) | [Humidity Sensor](#humidity) | [Rotation Vector Sensor](#rotation)  |
-| [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Light Sensor](#light) | [Sleep Monitor](#sleep_monitor)  |
-| [Gravity Sensor](#gravity)   | [Linear Acceleration Sensor](#lin_accelerometer) | [Temperature Sensor](#temperature)  |
-| [Gyroscope](#gyro)    | [Magnetic Sensor](#magnetic) | [Ultraviolet Sensor](#ultraviolet)  |
-| [Gyroscope Rotation Vector Sensor](#gyro_rotation)  | [Orientation Sensor](#orientation) | [Uncalibrated Gyroscope](#uncal_gyro)  |
-| [Heart Rate Monitor Sensor](#hrm)   | [Pedometer](#pedometer) | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
-| [Heart Rate Monitor Batch Sensor](#hrm_batch) | [Pressure Sensor](#pressure)  |  |
+| [Accelerometer](#accelerometer)  | [Heart Rate Monitor Batch Sensor](#hrm_batch) | [Proximity Sensor](#proximity)   |
```

Comment:
All the terms used here are using title-style capitalization. But in the file: docs/application/native/guides/location-sensors/device-sensors.md, all the terms are using sentence style capitalization. @mijin85cho requesting your input on this.

Reply:
After device-sensor pages for Native and .NET were set, we decided to use the sentence style to the titles.
Native had updates after the decision, so the sentence style was applied. 
So .NET is still in title-style, but Native is in sentence-style.

Reply:
I changed to use sentence-style in .Net page